### PR TITLE
[NUI] Update FontSizeScale values

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
### Description of Change ###
- Except for TV and wearable profiles, FontSizeScale values are updated.

### API Changes ###
- N/A

### CODE REVIEW

**Review by hinohie — APPROVED** [review]
Constant value sync patch. Looks good to me.

**Review by wonrst — APPROVED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs [hunk 1]
```diff
@@ -1137,11 +1137,11 @@ public static List<FontInfo> GetFontInfoList(PropertyArray fontArray)
...
 #else   // PROFILE_MOBILE and etc
         // The following values from 'system-settings/libutil/sstu.c'
-        private const float FontSizeScaleSmall = 0.8f;
+        private const float FontSizeScaleSmall = 0.87f;
         private const float FontSizeScaleNormal = 1.0f;
-        private const float FontSizeScaleLarge = 1.5f;
-        private const float FontSizeScaleHuge = 1.9f;
-        private const float FontSizeScaleGiant = 2.5f;
+        private const float FontSizeScaleLarge = 1.1f;
+        private const float FontSizeScaleHuge = 1.12f;
```
Comment (jaehyun0cho) [inline src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs:1143]: This should be 1.2f

**Review by jaehyun0cho — CHANGES_REQUESTED** [review]

File: src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs]: Thank you for checking the value! Fixed it!

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**